docs/content/v0.20/architecture.md

```diff
@@ -0,0 +1,36 @@
+# KubeStellar Architecture
+
+**NOTE**: This is only a very rudimentary start.
+
+## Higher Level
+
+At the higher level there are three modules.
+
+1. KubeFlex
+2. [the status addon](https://github.ibm.com/dettori/status-addon)
```

Comment:
the is implementation details, unless you are going to list here all the controllers, I would say the status addon is part of KubeStellar

Reply:
The point here is that the status addon is NOT in the KubeStellar repo, so at a high level --- where we look at repos, go modules, and dependencies --- we have to say that the status addon is currently separate. I think that it _should_ move into the KubeStellar repo, but right now we have to document the current state of affairs.

Reply:
We should definitely move it under the org https://github.com/kubestellar . As of moving into https://github.com/kubestellar/kubestellar - tried previously and found go deps issues could not resolve. But the main reason to keep it separate is modularity - we wanted to separate OCM - specific code in the transport-related repo(s).

Reply:
OK, so the future move is not entirely clear yet.

Reply:
Well, I propose to move the private repo to a new repo  https://github.com/kubestellar/status-addon - we just have to agree on that.